docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -49,22 +114,147 @@ GET /_search
 --------------------------------------------------
 // CONSOLE
 
-The query `path` points to the nested object path, and the `query`
-includes the query that will run on the nested docs matching the
-direct path, and joining with the root parent docs. Note that any
-fields referenced inside the query must use the complete path (fully
-qualified).
-
-The `score_mode` allows to set how inner children matching affects
-scoring of parent. It defaults to `avg`, but can be `sum`, `min`,
+Adding the option `score_mode` allows you to set how inner children matching
+affects scoring of parent. It defaults to `avg`, but can be `sum`, `min`,
 `max` and `none`.
 
-There is also an `ignore_unmapped` option which, when set to `true` will
-ignore an unmapped `path` and will not match any documents for this query.
-This can be useful when querying multiple indexes which might have different
-mappings. When set to `false` (the default value) the query will throw an
-exception if the `path` is not mapped.
+The `ignore_unmapped` option set to `true` will ignore an unmapped `path` and
+will not match any documents for this query. This can be useful when querying
+multiple indexes which might have different mappings. Set to `false` (the
+default value) the query will throw an exception if the `path` is not mapped.
+
```

Comment:
Present tense? Maybe "Setting the `ignore_unmapped` option to `true` ignores paths that don't exist in the nested field mapping, which can be useful when querying multiple indices with different mappings. By default, `ignore_unmapped` is set to `false` and an exception is thrown if the path is unmapped."